app/views/pages/items.js

```diff
@@ -70,6 +70,13 @@ let ItemsViewController = React.createClass({
     }
   },
 
+  componentWillReceiveProps: function(nextProps) {
+    if (this.getParams().id !== this.state.product.id) {
+      ProductActions.init(this.getParams().id);
```

Comment:
@wookiehangover this jogs my memory about the filter state get flushed out on the sidebar upon product init, Ill pull and see check if its all good